legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/domain/api/review/ReviewApi.java

```diff
@@ -66,18 +75,20 @@
      * @param assignedToMe whether to return only reviews assigned to me
      * @param authoredByMe whether to return only reviews authored by me
      * @param labels       labels to apply, return only reviews that match all the labels
+     * @param workspaceId workspace Id with which review is associated
+     * @param workspaceType workspace type with which review is associated
      * @param state        review state
      * @param since        this time limit is interpreted based on the chosen state, for example: if only committed reviews are fetched, 'since' will concern the committed time
      * @param until        this time limit is interpreted based on the chosen state, for example: if only committed reviews are fetched, 'since' will concern the committed time
      * @param limit        maximum number of reviews to get
      * @return reviews
      */
-    List<Review> getReviews(boolean assignedToMe, boolean authoredByMe, List<String> labels, ReviewState state, Instant since, Instant until, Integer limit);
+    List<Review> getReviews(boolean assignedToMe, boolean authoredByMe, List<String> labels, WorkspaceType workspaceType, Predicate<String> workspaceId, ReviewState state, Instant since, Instant until, Integer limit);
```

Comment:
```suggestion
    List<Review> getReviews(boolean assignedToMe, boolean authoredByMe, List<String> labels, Predicate<String> workspaceId, Set<WorkspaceType> workspaceTypes, ReviewState state, Instant since, Instant until, Integer limit);
```